Sage/carbon
# fix(simple-select): prevent select list from opening via enter key when component is focused

### Proposed behaviour



Users can no longer press the Enter key to open the select list in `SimpleSelect`

### Current behaviour



Currently, when the component is focused and a user presses the enter key, the component opens. This is not aligned with the default behaviour of the native select element and has been removed.

### Checklist




#### QA


### Additional context



### Testing instructions



Make sure all tests pass, and any tests (skipped or un-skipped) relating to the enter key opening the select list in `SimpleSelect` or components which render `SimpleSelect` are removed
